src/vaadin-date-time-picker-date-text-field.html

```diff
@@ -6,6 +6,32 @@
 
 <link rel="import" href="../../vaadin-text-field/src/vaadin-text-field.html">
 
+<dom-module id="vaadin-date-time-picker-date-text-field-styless" theme-for="vaadin-date-time-picker-date-text-field">
+  <template>
+    <style>
+      :host([dir="rtl"]) [part="input-field"] {
```

Comment:
Wouldn't it make more sense to import `<link rel="import" href="../../vaadin-date-picker/src/vaadin-date-picker-text-field.html">` and then something like:
```js
class DateTimePickerDateTextFieldElement extends customElements.get('vaadin-date-picker-text-field')
```
Then we wouldn't need to duplicate the RTL styles here.

Reply:
Also noticed there's a typo in `-styless"`